src/types.ts

```diff
@@ -15,11 +15,6 @@ export interface EventObject {
    * The type of event that is sent.
    */
   type: string;
-  /**
-   * The unique ID that identifies this specific event instance.
-   */
-  id?: string | number;
```

Comment:
don't think `id` has currently any meaning on the event object as that thing (sendid) is set on SendAction and SCXML.Event